### PR TITLE
Add CloudIngressOperatorOfflineSRE alert

### PR DESCRIPTION
**Pull request by sam-nguyen7**
Add alert for when cloud-ingress-operator metrics service is not found

### CODE REVIEW

File: deploy/sre-prometheus/100-cloud-ingress-operator.PrometheusRule.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+apiVersion: monitoring.coreos.com/v1
+￼kind: PrometheusRule
```
Comment (cblecker) [inline deploy/sre-prometheus/100-cloud-ingress-operator.PrometheusRule.yaml:2]: You have some hidden characters in here that is causing the generated SSS to break below

Comment (sam-nguyen7) [reply deploy/sre-prometheus/100-cloud-ingress-operator.PrometheusRule.yaml]: ah thank you for catching that!

File: deploy/sre-prometheus/100-cloud-ingress-operator.PrometheusRule.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    prometheus: sre-cloud-ingress-operator-offline-alerts
+    role: alert-rules
+  name: sre-cloud-ingress-operator-offline-alerts
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-cloud-ingress-operator-offline-alerts
+    rules:
+    - alert: CloudIngressOperatorOfflineSRE
+      expr: absent(up{service="localmetrics-cloud-ingress-operator"})
```
Comment (cblecker) [inline deploy/sre-prometheus/100-cloud-ingress-operator.PrometheusRule.yaml:14]:
```suggestion
      expr: absent(up{service="localmetrics-cloud-ingress-operator", namespace="openshift-cloud-ingress-operator"})
```

**Comment by cblecker** [comment]
Any update on this @sam-nguyen7 ?

**Comment by cblecker** [comment]
/hold cancel
SOP in https://github.com/openshift/ops-sop/pull/791

**Comment by cblecker** [comment]
Note that this should not be promoted to production until https://issues.redhat.com/browse/OSD-3902 is addressed

**Pull request merged**